ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py

```diff
@@ -368,3 +368,35 @@ def test_torch_svd(
         atol=1e-2,
         ground_truth_backend=frontend,
     )
+    
+    
+#svdvals
+@handle_frontend_test(
+    fn_tree="torch.linalg.svdvals",
+    dtype_and_x=_get_dtype_and_square_matrix(),
+    n=helpers.ints(min_value=2, max_value=5),
```

Comment:
You've sampled an argument `n`, but haven't passed it in the test function. Also as far as this function is concerned, there doesn't exist any `n` keyword argument in it's implementation. Kindly remove it, thanks !

Reply:
I have removed the argument. Please review. Thanks !